crates/oxc_linter/src/rules/eslint/func_names.rs

```diff
@@ -142,6 +156,34 @@ fn is_object_or_class_method(parent_node: &AstNode) -> bool {
     }
 }
 
+// enum InferredName<'a> {
```

Comment:
It looks like do the same as `guess_function_name`, is `InferedName` preparing for the future work?

Reply:
Oops forgot to remove that, thanks!